plugin.rb

```diff
@@ -9,6 +9,7 @@
 
 %i[common mobile].each do |type|
   register_asset "stylesheets/#{type}/question-answer.scss", type
+  register_asset "stylesheets/#{type}/question-answer-crawler.scss", type
```

Comment:
It looks like we only have a `common` stylesheet so registering the mobile stylesheet seems incorrect?